cli/how-to-deploy-triton-ensemble-managed-online-endpoint.sh

```diff
@@ -0,0 +1,49 @@
+## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
+## Please reach out to the Azure ML docs & samples team before before editing for the first time.
+
+set -e
+
+# <set_endpoint_name>
+export ENDPOINT_NAME=triton-ensemble-mir-endpt-`echo $RANDOM`
+# </set_endpoint_name>
+
+# <deploy>
+az ml endpoint create -n $ENDPOINT_NAME -f endpoints/online/triton/ensemble/create-endpoint-with-deployment-mir.yml
+# </deploy>
+
+# <get_status>
+az ml endpoint show -n $ENDPOINT_NAME
+# </get_status>
+
+#  check if create was successful
+endpoint_status=`az ml endpoint show --name $ENDPOINT_NAME --query "provisioning_state" -o tsv`
+echo $endpoint_status
+if [[ $endpoint_status == "Succeeded" ]]
+then  
+  echo "Endpoint created successfully"
+else
+  echo "Endpoint creation failed"
+  exit 1
+fi
+
+deploy_status=`az ml endpoint show --name $ENDPOINT_NAME --query "deployments[?name=='blue'].provisioning_state" -o tsv`
+echo $deploy_status
+if [[ $deploy_status == "Succeeded" ]]
+then
+  echo "Deployment completed successfully"
+else
+  echo "Deployment failed"
+  exit 1
+fi
+
+# <get_logs>
+az ml endpoint get-logs -n $ENDPOINT_NAME --deployment blue
+# </get_logs>
+
+# <get_scoring_uri>
+az ml endpoint show -n $ENDPOINT_NAME --query "scoring_uri"
+# </get_scoring_uri>
```

Comment:
@Rupsan123 we need to show scoring either with `az ml endpoint invoke` or with `curl`

Reply:
Added scoring to check server status.
Need to add actual scoring.